block_data/shorthands.tex

```diff
@@ -0,0 +1,123 @@
+We shall often use the following shorthands for
+\[
+	\left\{ \begin{array}{lcl}
+		\previousConflation & \longleftrightarrow & \locIsPrevConf   \\
+		\currentConflation  & \longleftrightarrow & \locIsCurrConf   \\
+		\isCoinbase         & \longleftrightarrow & \locIsCoinbase   \\
+		\isTimestamp        & \longleftrightarrow & \locIsTimestamp  \\
+		\isNumber           & \longleftrightarrow & \locIsNumber     \\
+		\isDifficulty       & \longleftrightarrow & \locIsDifficulty \\
+		\isGaslimit         & \longleftrightarrow & \locIsGaslimit   \\
+		\isChainid          & \longleftrightarrow & \locIsChainid    \\
+		\isBasefee          & \longleftrightarrow & \locIsBasefee    \\
+	\end{array} \right.
```

Comment:
I suggest to defined those as aliases of columns names using: https://www.notion.so/consensys/Naming-Conventions-55b293749c9a459eaef5c96d7f982267?pvs=4#91deab67ed894ae78b1311beb8380443.